purchase_order_vendor_product/views/purchase_order.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="purchase_order_form_vendor_wise_product" model="ir.ui.view">
+        <field name="name">purchase.order.form.inherit.vendor.wise.product</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//tree/field[@name='product_id']" position="attributes">
```

Comment:
this should target the tree inside the lines field to avoid clashes if someone adds another tree to the form, no?

Reply:
> this should target the tree inside the lines field to avoid clashes if someone adds another tree to the form, no?

Yup looks reasonable. Will fix